src/common/com/intellij/plugins/haxe/lang/psi/impl/AbstractHaxePsiClass.java

```diff
@@ -82,6 +83,12 @@ public String getQualifiedName() {
     return HaxeResolveUtil.joinQName(packageName, name);
   }
 
+  private HaxeClassModel _model = null;
+  public HaxeClassModel getModel() {
+    if (_model == null) _model = new HaxeClassModel(this);
```

Comment:
I will try, but since our coding standard is to use single line for small statements like guard clauses and lazy initializations maybe I miss some of them. So better to just apply a code reformatting over the code tree after the merge when this is ready, so all the code is normalizing following the coding standards defined in the project.